From 1b128432e354b8d9277872f7679c1ac439cb781b Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Fran=C3=A7ois=20Revol?= <revol@free.fr>
Date: Sun, 13 Nov 2016 03:15:05 +0100
Subject: Disable Fortran support for now

We don't really build gcc with the fortran frontend yet...

diff --git a/CMakeLists.txt b/CMakeLists.txt
index e79ad41..3455ab7 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -39,8 +39,8 @@ else()
 endif()
 
 enable_language(C)
-enable_language (Fortran)
-set(NOFORTRAN FALSE)
+#enable_language (Fortran)
+#set(NOFORTRAN FALSE)
 set(SUPERLU_VERSION "${PROJECT_VERSION}")
 set(SUPERLU_REV "${PROJECT_REV}")
 
-- 
2.7.0

